Resfeber/CoreData/CoreDataStack.swift

```diff
@@ -10,63 +10,64 @@ import Foundation
 import CoreData
 
 open class CoreDataStack {
-  public static let modelName = "Trip"
+    public static let modelName = "Trip"
+    static let sharedCoreDataStack = CoreDataStack()
```

Comment:
just a nitpick, but I would just call this `shared`

It's somewhat redundant to have to call `CoreDataStack.sharedCoreDataStack`. Instead you could just call `CoreDataStack.shared` and you really don't lose any meaning